pkg/validation/specialized_test.go

```diff
@@ -11,35 +11,6 @@ import (
 )
 
 var _ = Describe("Specialized", func() {
-	Context("With invvalid data", func() {
-
-		var (
-			vmCirros *k6tv1.VirtualMachine
-			vmRef    *k6tv1.VirtualMachine
-		)
-
-		BeforeEach(func() {
-			vmCirros = NewVMCirros()
-			vmRef = k6tobjs.NewDefaultVirtualMachine()
-		})
-
-		It("Should detect bogus rules", func() {
-			r := validation.Rule{
-				Rule:    "integer-value",
-				Name:    "EnoughMemory",
-				Path:    "jsonpath::.spec.domain.resources.requests.memory",
-				Message: "Memory size not specified",
-				Valid:   "jsonpath::.spec.domain.this.path.does.not.exist",
-				Min:     64 * 1024 * 1024,
-				Max:     512 * 1024 * 1024,
-			}
-
-			ra, err := r.Specialize(vmCirros, vmRef)
-			Expect(err).To(Not(BeNil()))
-			Expect(ra).To(BeNil())
-		})
-	})
-
 	Context("With valid data", func() {
 
```

Comment:
Could you please add some positive tests for the `Valid` field?

Reply:
There are already tests for `Valid` in `pkg/validation/eval_test.go`.